tests/decodecorpus.c

```diff
@@ -1392,7 +1394,7 @@ static int runTestMode(U32 seed, unsigned numFiles, unsigned const testDurationS
 
     for (fnum = 0; fnum < numFiles || clockSpan(startClock) < maxClockSpan; fnum++) {
         frame_t fr;
-
+        U32 const seedCopy = seed;
```

Comment:
OK, looks good.
So, if I understand properly,
`seed` is used (and modified) only once per round,
to generate a frame.

By displaying the current round's `seed`,
it's possible to jump directly to said `seed` in future tests, using `-s#` command,
in order to reproduce an error case.

That would make `fnum` non-essential, since it's not a useful information to jump directly to the wanted test case.